sql/core/benchmarks/ConstantColumnVectorBenchmark-jdk21-results.txt

```diff
@@ -1,280 +1,280 @@
-ConstantColumnVector                                           1952           1956           6        209.9           4.8       1.0X
-OnHeapColumnVector                                             5742           5745           3         71.3          14.0       0.3X
-OffHeapColumnVector                                            8555           8574          26         47.9          20.9       0.2X
+ConstantColumnVector                                           1960           1961           1        209.0           4.8       1.0X
+OnHeapColumnVector                                             5695           5696           1         71.9          13.9       0.3X
+OffHeapColumnVector                                            8553           8553           0         47.9          20.9       0.2X
 
-OpenJDK 64-Bit Server VM 21.0.5+11-LTS on Linux 6.8.0-1017-azure
+OpenJDK 64-Bit Server VM 21.0.6+7-LTS on Linux 6.8.0-1020-azure
 AMD EPYC 7763 64-Core Processor
 Test write and read with StringType, row length = 30:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------------------
-ConstantColumnVector                                           1956           1957           0        209.4           4.8       1.0X
-OnHeapColumnVector                                             5657           5661           4         72.4          13.8       0.3X
-OffHeapColumnVector                                            8523           8539          23         48.1          20.8       0.2X
+ConstantColumnVector                                           1957           1960           5        209.3           4.8       1.0X
+OnHeapColumnVector                                             5699           5708          14         71.9          13.9       0.3X
+OffHeapColumnVector                                            8550           8558          11         47.9          20.9       0.2X
 
-OpenJDK 64-Bit Server VM 21.0.5+11-LTS on Linux 6.8.0-1017-azure
+OpenJDK 64-Bit Server VM 21.0.6+7-LTS on Linux 6.8.0-1020-azure
 AMD EPYC 7763 64-Core Processor
 Test write and read with IntegerType:     Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-ConstantColumnVector                                892            892           1        459.3           2.2       1.0X
-OnHeapColumnVector                                 1020           1021           1        401.5           2.5       0.9X
-OffHeapColumnVector                                 892            893           1        459.0           2.2       1.0X
+ConstantColumnVector                                  0              0           0   13274135.5           0.0       1.0X
```

Comment:
This looks strange because the value is `0`.

Reply:
The 0s are strange. Do we also get 0s for master branch for this benchmark result?